sky/serve/controller.py

```diff
@@ -88,7 +88,12 @@ def _run_autoscaler(self):
                              f'{common_utils.format_exception(e)}')
                 with ux_utils.enable_traceback():
                     logger.error(f'  Traceback: {traceback.format_exc()}')
-            time.sleep(constants.AUTOSCALER_DEFAULT_DECISION_INTERVAL_SECONDS)
+            if self._autoscaler.target_num_replicas == 0:
+                time.sleep(
+                    constants.AUTOSCALER_NO_REPLICA_DECISION_INTERVAL_SECONDS)
+            else:
+                time.sleep(
+                    constants.AUTOSCALER_DEFAULT_DECISION_INTERVAL_SECONDS)
```

Comment:
Why we need this?

Reply:
@cblmemo Reduce the autoscaler interval when there is no replica - for quicker scale up if `min_replicas=0`.

Reply:
Could we make this an autoscaler member function (which returns the next interval time)?

Reply:
sounds good.